scoreboard-colors.scss

```diff
@@ -113,104 +113,108 @@ $color-badge-disabled-border: rgba($color-badge-border, .5);
+$palette-yellow-light:             #FBE499;
+
+$palette-green-dark:               #619900;
+$palette-green-medium:             #96CD00;
+$palette-green-light:              #CAE779;
+
+$palette-turquoise-dark:           #00A9C3;
+$palette-turquoise-medium:         #60CBDB;
+$palette-turquoise-light:          #CCF3f9;
+
+$palette-blue-dark:                #0074BB;
+$palette-blue-medium:              #1A94DA;
+$palette-blue-light:               #CDE9FA;
+
+$palette-periwinkle-dark:          #3654D4;
+$palette-periwinkle-medium:        #839AFF;
+$palette-periwinkle-light:         #C4CFFF;
+
+$palette-purple-dark:              #6D1DB8;
+$palette-purple-medium:            #AD5AFA;
+$palette-purple-light:             #DBB5FF;
+
+$palette-fuchsia-dark:             #A31DB8;
+$palette-fuchsia-medium:           #E55BFA;
+$palette-fuchsia-light:            #F5B5FF;
+
+$palette-gray-darker:              $color-black;
+$palette-gray-dark:                #8E9699;
+$palette-gray-medium:              #BCC2C4;
+$palette-gray-light:               #D7DEE0;
+
+$palette-gray:                     #EDEDED;
+$palette-slate:                    #8E9699;
+$palette-infantry-blue:            #ABBFD8;
+$palette-white:                    $color-white;
+
+$palette-initiative-budget:        #2164bd;
+$palette-initiative-schedule:      #8977b2;
+$palette-lighter-gray:             #f3f5f6;
+
+$palette-colors:
+  red-dark $palette-red-dark,
+  red-medium $palette-red-medium,
+  red-light $palette-red-light,
+
+  orange-dark $palette-orange-dark,
+  orange-medium $palette-orange-medium,
+  orange-light $palette-orange-light,
+
+  yellow-dark $palette-yellow-dark,
+  yellow-medium $palette-yellow-medium,
+  yellow-light $palette-yellow-light,
+
+  green-dark $palette-green-dark,
+  green-medium $palette-green-medium,
+  green-light $palette-green-light,
+
+  turquoise-dark $palette-turquoise-dark,
+  turquoise-medium $palette-turquoise-medium,
+  turquoise-light $palette-turquoise-light,
+
+  blue-dark $palette-blue-dark,
+  blue-medium $palette-blue-medium,
+  blue-light $palette-blue-light,
+
+  periwinkle-dark $palette-periwinkle-dark,
+  periwinkle-medium $palette-periwinkle-medium,
+  periwinkle-light $palette-periwinkle-light,
+
+  purple-dark $palette-purple-dark,
+  purple-medium $palette-purple-medium,
+  purple-light $palette-purple-light,
+
+  fuchsia-dark $palette-fuchsia-dark,
+  fuchsia-medium $palette-fuchsia-medium,
+  fuchsia-light $palette-fuchsia-light,
+
+  gray-darker $palette-gray-darker,
+  gray-dark $palette-gray-dark,
+  gray-medium $palette-gray-medium,
+  gray-light $palette-gray-light,
+
+  gray $palette-gray,
+  slate $palette-slate,
+  infantry-blue $palette-infantry-blue,
+  white $palette-white
+
+  initiative-budget $palette-initiative-budget,
+  initiative-schedule $palette-initiative-schedule,
+  lighter-gray $palette-lighter-gray;
 
 // These aren't used anywhere yet. They're for colored cells where the text is a darker
```

Comment:
I asked a comment in slack regarding this comment the commented out code below it. It's from 2018 and I think it causes more confusion than it's worth. It's pretty dated, can we delete the commented out code? @scottoreilly You wrote this block.

Reply:
Absolutely, let’s remove commented out stuff.